command/sub8_alarm/alarm_handlers/network_loss.py

```diff
@@ -6,7 +6,7 @@ class NetworkLoss(HandlerBase):
     alarm_name = 'network-loss'
 
     def __init__(self):
-        self.hm = HeartbeatMonitor(self.alarm_name, "/network", Header, node_name="network_loss_kill")
+        self.hm = HeartbeatMonitor(self.alarm_name, "/network", node_name="network_loss_kill")
```

Comment:
```
class HeartbeatMonitor(AlarmBroadcaster):
    def __init__(self, alarm_name, topic_name, msg_class, prd=0.2, predicate=None, nowarn=False, **kwargs):
        ''' Used to trigger an alarm if a message on the topic `topic_name` isn't published
            atleast every `prd` seconds.
```
I changed it, because this was the definition of HeartbeatMonitor I had locally and it caused an error. I guess I should pull more often...